manager/src/main/java/com/redhat/service/bridge/manager/api/user/validators/actions/ActionParamValidatorContainer.java

```diff
@@ -34,7 +34,7 @@ public boolean isValid(ProcessorRequest value, ConstraintValidatorContext contex
             return false;
         }
 
-        if (baseAction.getParameters() == null || baseAction.getParameters().isEmpty()) {
```

Comment:
@kostola Does removing the `.isEmpty()` check here impact on the already existing validators for `SendToKafka` and `Webhook` Actions? For example, the `SendToKafka` action validation should fail if there are no parameters.

Reply:
This check is done before calling the action-specific validator. Now the common validation part tolerates actions with no parameters.
The `SendToKafka` and `Webhook` specific validators already do a further check if their expected parameter is there and is not empty and fail otherwise, so the end behavior is not altered by this change.

Reply:
Perfect. Cheers 👍